tests/unit/browsers/test_browser_pool.py

```diff
@@ -151,3 +151,12 @@ async def test_methods_raise_error_when_not_active() -> None:
 
     async with browser_pool:
         assert browser_pool.active is True
+
+
+async def test_with_plugin_contains_page_options(httpbin: URL) -> None:
+    plugin = PlaywrightBrowserPlugin(page_options={'user_agent': 'My Best Uset-Agent'})
```

Comment:
Super tiny irrelevant typo: My Best Use**t**-Agent

Reply:
Thank you for noticing that!